distribution/docker/build.gradle

```diff
@@ -162,7 +162,7 @@ void addBuildDockerImage(final boolean oss) {
       ]
     }
     executable 'docker'
-    final List<String> dockerArgs = ['build', files(oss), '--pull', '--no-cache']
+    final List<String> dockerArgs = ['build', files(oss), '--network', 'host', '--pull', '--no-cache']
```

Comment:
Why are we switching to host networking here? This has different semantics on a Linux machine with "native" docker vs Docker-for-desktop type scenario, like on OSX where the docker engine is running inside a VM.

Reply:
I think that's a mistake, I've removed it.